### PR TITLE
Explanation about guest users on MT apps

### PR DESCRIPTION
**Pull request by TiagoBrenck**
## Purpose
Added an explanation about `/common` endpoint and guest users.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: 2-WebApp-graph-user/2-3-Multi-Tenant/README.md [hunk 1]
```diff
@@ -216,6 +216,10 @@ services.AddAuthentication(AzureADDefaults.AuthenticationScheme)
 
  You can read about the various endpoints of the Microsoft Identity Platform [here](https://docs.microsoft.com/azure/active-directory/develop/active-directory-v2-protocols#endpoints).
 
+#### Implications of signing-in guest users on /common endpoint
+
+Please note that if you sign-in guest users at the `/common` endpoint, they will be directed to their home tenant for signing-in. So, if your multi-tenant app cares about applying a tenant specific CA policies, group assignments or app roles to be applied to the guest users, the app should sign the guest user on the **tenanted endpoint** (https://login.microsoftonline.com/{tenantId}) instead of the `/common` endpoint.
```
Comment (pmaytak) [inline 2-WebApp-graph-user/2-3-Multi-Tenant/README.md]: Should 'CA' be spelled out as 'Conditional Access'? I think generally an abbreviation should be spelled out the first time it's used in the text unless it's common.

Comment (TiagoBrenck) [reply 2-WebApp-graph-user/2-3-Multi-Tenant/README.md]: fixed

File: 2-WebApp-graph-user/2-3-Multi-Tenant/README-National-Cloud.md [hunk 2]
```diff
@@ -240,6 +240,10 @@ services.AddAuthentication(AzureADDefaults.AuthenticationScheme)
 
  You can read about the various endpoints of the Microsoft Identity Platform [here](https://docs.microsoft.com/azure/active-directory/develop/active-directory-v2-protocols#endpoints).
 
+#### Implications of signing-in guest users on /common endpoint
+
+Please note that if you sign-in guest users at the `/common` endpoint, they will be directed to their home tenant for signing-in. So, if your multi-tenant app cares about applying a tenant specific conditional access policies, group assignments or app roles to be applied to the guest users, the app should sign the guest user on the **tenanted endpoint** (https://login.microsoftonline.com/{tenantId}) instead of the `/common` endpoint.
```
Comment (jennyf19) [inline 2-WebApp-graph-user/2-3-Multi-Tenant/README-National-Cloud.md]: remove "a"

Comment (jennyf19) [inline 2-WebApp-graph-user/2-3-Multi-Tenant/README-National-Cloud.md]: >sign [](start = 307, length = 4)

should "sign-in" the guest user

File: 2-WebApp-graph-user/2-3-Multi-Tenant/README.md [hunk 3]
```diff
@@ -216,6 +216,10 @@ services.AddAuthentication(AzureADDefaults.AuthenticationScheme)
 
  You can read about the various endpoints of the Microsoft Identity Platform [here](https://docs.microsoft.com/azure/active-directory/develop/active-directory-v2-protocols#endpoints).
 
+#### Implications of signing-in guest users on /common endpoint
+
+Please note that if you sign-in guest users at the `/common` endpoint, they will be directed to their home tenant for signing-in. So, if your multi-tenant app cares about applying a tenant specific conditional access policies, group assignments or app roles to be applied to the guest users, the app should sign the guest user on the **tenanted endpoint** (https://login.microsoftonline.com/{tenantId}) instead of the `/common` endpoint.
```
Comment (jennyf19) [inline 2-WebApp-graph-user/2-3-Multi-Tenant/README.md]: > a [](start = 179, length = 2)

remove "a"

Comment (jennyf19) [inline 2-WebApp-graph-user/2-3-Multi-Tenant/README.md]: >sign [](start = 307, length = 4)

"sign-in"

**Comment by jennyf19** [comment]
@tiagobrenck this is super! can we put this wording somewhere in the MSAL repo as well? it's very clear. well done.

---
In reply to: [362081420](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/pull/306#issuecomment-362081420) [](ancestors = 362081420)

**Pull request merged**